### DISCUSSION TITLE
Supporting to skin a Main Menu as skinning layout

### DISCUSSION THREAD

**Discussion by LunaticalFrost** [ppy/osu#19600] [Ideas] [answered]
**I think that this idea fits in well with future changes in the skins of the game. Some people playing the stable version were, and still are, familiar with the legacy main menu interface. And personally, if I wanted to prefer - staying with the legacy menu interface or the one we have now - I'd choose both and/or preferably make it the player's choice, or make the main menu buttons a fully customizable element to change for absolute convenience like on gameplay, for example.** If there is possible to realize that.

**Comment by peppy** [comment, marked as answer]
Out of scope sorry.

The main menu design will change going forward, so don't worry too much. You'll have some options.

**Reply by LunaticalFrost** [reply]
Thanks for that)
